plugins/talk-plugin-auth/client/components/ForgotContent.js

```diff
@@ -20,13 +33,13 @@ class ForgotContent extends React.Component {
         </div>
         <form onSubmit={this.handleSubmit}>
           <div className={styles.textField}>
-            <label htmlFor="email">{t('sign_in.email')}</label>
-            <input
-              ref={(input) => (this.emailInput = input)}
-              type="text"
+            <TextField
+              type="email"
               style={{fontSize: 16}}
               id="email"
               name="email"
+              label={t('sign_in.email')}
+              onChange={this.handleChangEmail}
```

Comment:
Let's also sync back `state.value` to the `TextField` via `value={this.state.value}`, to make sure that they are always stay in sync.